packages/backend/src/config/Config.ts

```diff
@@ -55,6 +55,7 @@ export interface ClockConfig {
 export interface TvlConfig {
   readonly alchemyApiKey: string
   readonly etherscanApiKey: string
+  readonly arbiscanApiKey: string
```

Comment:
This is obviously good for now, but do you have any ideas how it could be handled in the future? I suppose we will have many configurations. This is just an open question and is not a part of this PR